src/Modules/Pages/Domain/RevisionBlock/RevisionBlockType.php

```diff
@@ -71,7 +71,17 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'class' => Block::class,
                 'required' => false,
                 'group_by' => static fn(Block $block): TranslationKey => $block->getBlock()->getModule()->asLabel(),
-                'choice_label' => static fn(Block $block): TranslationKey => $block->getLabel(),
+                'choice_label' => function (Block $block): string|TranslationKey {
```

Comment:
wrong module, should be implemented in the ContentBlocks module

Reply:
How do you mean? This was done so the labels for the ContentBlocks widgets on the page builder would be the block's title, the way it works now, and not all 'lbl.ContentBlock'. Can I override the behaviour of the page builder in the ContentBlocks module?

Reply:
Check how this was done in fork5, you should first check if there is a setting with the name label_LANGUAGECODE if not check if there is a setting with the name label, and otherwise fall back to the default, but all of that needs to be done in Block::getLabel

Reply:
Using trans() now for the RevisionBlock labels